src/modules/keyboardmanager/KeyboardManagerEditor/Resources.resx

```diff
@@ -149,24 +149,24 @@
     <value>Cancel</value>
   </data>
   <data name="Continue_Button" xml:space="preserve">
-    <value>Continue Anyway</value>
+    <value>Continue anyway</value>
   </data>
   <data name="EditKeyboard_SourceHeader" xml:space="preserve">
-    <value>Physical Key:</value>
-    <comment>Key on a keyboard</comment>
+    <value>Press:</value>
+    <comment>verb</comment>
   </data>
   <data name="EditKeyboard_TargetHeader" xml:space="preserve">
-    <value>Mapped To:</value>
+    <value>To send:</value>
```

Comment:
You don’t “send to” this language would cause the same problem we have now

Reply:
Well then how should we name it? I've had my issue open for 12 months now, but if nobody answers...?

Reply:
P.S. nowhere does it say "send to'"

Reply:
It says "To send:"
The new formulation might be clearer than the current one.
@crutkas , what do you think?

Reply:
Press - To send sound intuitive to me and better than what we have now